packages/core/package.json

```diff
@@ -61,6 +60,7 @@
         "@ai16z/adapter-sqlite": "workspace:*",
         "@ai16z/adapter-sqljs": "workspace:*",
         "@ai16z/adapter-supabase": "workspace:*",
+        "@ai16z/plugin-solana": "workspace:*",
```

Comment:
We can't have this reference here, this test sould be moved to the solana plugin